libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/GranularPermissions/GranularPermissionsDropdown.tsx

```diff
@@ -70,7 +91,15 @@ export const GranularPermissionsDropdown: React.FC<IGranularPermissionsDropdownP
                     },
                 )}
                 onClick={handleClick}
-                aria-label="Share dialog granular permissions button"
+                onKeyDown={handleKeyDown}
+                aria-label={intl.formatMessage({ id: "shareDialog.share.granular.grantee.permission.label" })}
+                aria-haspopup
```

Comment:
can this custom dropdown be rewritten to our Dropdown with its button and body props? It should have opening/closing of body already solved

Reply:
Changed to use our Dropdown